kpops/components/streams_bootstrap_v3/streams/model.py

```diff
@@ -0,0 +1,266 @@
+
+    :param enabled: Whether to enable auto-scaling using KEDA., defaults to False
+    :param lag_threshold: Average target value to trigger scaling actions.
+        Mandatory to set when auto-scaling is enabled.
+    :param polling_interval: This is the interval to check each trigger on.
+        https://keda.sh/docs/2.9/concepts/scaling-deployments/#pollinginterval,
+        defaults to 30
+    :param cooldown_period: The period to wait after the last trigger reported
+        active before scaling the resource back to 0.
+        https://keda.sh/docs/2.9/concepts/scaling-deployments/#cooldownperiod,
+        defaults to 300
+    :param offset_reset_policy: The offset reset policy for the consumer if the
+        consumer group is not yet subscribed to a partition.,
+        defaults to "earliest"
+    :param min_replicas: Minimum number of replicas KEDA will scale the resource down to.
+        "https://keda.sh/docs/2.9/concepts/scaling-deployments/#minreplicacount",
+        defaults to 0
+    :param max_replicas: This setting is passed to the HPA definition that KEDA
+        will create for a given resource and holds the maximum number of replicas
+        of the target resouce.
+        https://keda.sh/docs/2.9/concepts/scaling-deployments/#maxreplicacount,
+        defaults to 1
+    :param idle_replicas: If this property is set, KEDA will scale the resource
+        down to this number of replicas.
+        https://keda.sh/docs/2.9/concepts/scaling-deployments/#idlereplicacount,
+        defaults to None
+    :param topics: List of auto-generated Kafka Streams topics used by the streams app.,
+        defaults to []
+    """
+
+    enabled: bool = Field(
+        default=False,
+        description=describe_attr("streams", __doc__),
+    )
+    lag_threshold: int | None = Field(
+        default=None,
+        title="Lag threshold",
+        description=describe_attr("lag_threshold", __doc__),
+    )
+    polling_interval: int = Field(
+        default=30,
+        title="Polling interval",
+        description=describe_attr("polling_interval", __doc__),
+    )
+    cooldown_period: int = Field(
+        default=300,
+        title="Cooldown period",
+        description=describe_attr("cooldown_period", __doc__),
+    )
+    offset_reset_policy: str = Field(
+        default="earliest",
+        title="Offset reset policy",
+        description=describe_attr("offset_reset_policy", __doc__),
+    )
+    min_replicas: int = Field(
+        default=0,
+        title="Min replica count",
+        description=describe_attr("min_replicas", __doc__),
+    )
+    max_replicas: int = Field(
+        default=1,
+        title="Max replica count",
+        description=describe_attr("max_replicas", __doc__),
+    )
+    idle_replicas: int | None = Field(
+        default=None,
+        title="Idle replica count",
+        description=describe_attr("idle_replicas", __doc__),
+    )
+    topics: list[str] = Field(
```

Comment:
What about internal topics?

Reply:
This should be renamed to `internal_topics`.

Reply:
No, there is both

Reply:
> No, there is both

Edited the message :D